app/Policies/EmployeePolicy.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace App\Policies;
+
+use App\Models\Employee;
+use App\Models\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class EmployeePolicy
```

Comment:
ini kyk e nanti blm dipake, karena terlalu rumit jadi folder policies e dihapus ae daripada nambah"in folder di `/app`

Reply:
iki pisan @Zaki140502